spec/integration/navigation_spec.rb

```diff
@@ -1,9 +1,9 @@
-require 'test_helper'
-
-class NavigationTest < ActionDispatch::IntegrationTest
+#require 'test_helper'
+#
+#class NavigationTest < ActionDispatch::IntegrationTest
```

Comment:
Incorrect indentation detected (column 0 instead of 2).<br>Missing space after #.